tool/lib/utils.dart

```diff
@@ -88,51 +88,52 @@ class CliCommand {
   final bool throwOnException;
 }
 
-Future<String> runProcess(
-  ProcessManager processManager,
-  CliCommand command, {
-  String? workingDirectory,
-  String? additionalErrorMessage = '',
-}) async {
-  String stdout = '';
+class DevToolsProcessManager {
+  final processManager = ProcessManager();
```

Comment:
we shouldn't need a class for this. These methods can be extension methods on ProcessManager

```dart
extension DevToolsProcessManagerExtension on ProcessManager {
  Future<String> runProcess(...) {}
  ...
}
```